### PR TITLE
Add new change types.

### PR DESCRIPTION
**Pull request by stealthycoin**
cc @dstufft @JordonPhillips @jamesls @kyleknap

### CODE REVIEW

File: scripts/new-change [hunk 1]
```diff
@@ -54,6 +54,10 @@ CHANGES_DIR = os.path.join(
 )
 TEMPLATE = """\
 # Type should be one of: feature, bugfix, enhancement, api-change
+# feature: A larger feature or change in behavior
```
Comment (kyleknap) [inline scripts/new-change]: It would probably be good to add something  long the lines of:
``which typically entails a minor version bump.``

Comment (stealthycoin) [inline scripts/new-change]: Sounds good.

File: scripts/new-change [hunk 2]
```diff
@@ -53,7 +53,12 @@ CHANGES_DIR = os.path.join(
     '.changes'
 )
 TEMPLATE = """\
-# Type should be one of: feature, bugfix
+# Type should be one of: feature, bugfix, enhancement, api-change
+# feature: A larger feature or change in behavior, usually resulting in a
+#          minor version bump
```
Comment (kyleknap) [inline scripts/new-change]: Sorry. Super nit picky but probably add a period at the end to make it consistent with the others. Or you can remove the periods.

**Comment by kyleknap** [comment]
Looks good. Go ahead and use this for the rest of your PR's and those should be good as well.

**Pull request merged**